calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -277,6 +287,8 @@ Before creating an exception, it is important to understand the effects to Image
 
   Pods could be created where they were previously blocked. Admission controller policies are based on vulnerability information (scan result and/or maximum CVSS score).
 
+To view your existing vulnerability exceptions, navigate to the Exceptions page. In the left navbar in Manager, click **Image Assurance**, and then **Vulnerability Exceptions**.
```

Comment:
I'll take a look around the other docs and see.

Reply:
Seems like navbar is used quite a bit: 
https://github.com/tigera/docs/search?q=navbar

The alternative would be just to spell it out "left-hand navigation menu". But because the rest of the docs site already uses navbar, it feels like something we have decided is acceptable already. So there is a case to be made for staying consistent.

Reply:
We do use the word 'navbar', for better or worse. If this needs changing, the docs team can handle this globally. Fine to keep it here for now.